test/unit/report_test.py

```diff
@@ -110,3 +115,42 @@ def test_static_code_analysis(
     coverage_report = named_temp_file(name=".lint.txt", content=content)
     actual = _static_code_analysis(coverage_report)
     assert actual == expected
+
+
+@pytest.mark.parametrize(
+    "lower,compare",
+    [
+        (["HH", "LL"], 0),
+        (["HM", "LM", "ML"], 0),
+        (["HL", "MH"], 0),
+        ([], 6),
+        (["HL"], ["MH"]),
+        (["MH"], ["MM"]),
+        (["MM"], ["ML"]),
+        (["HL"], ["LL"]),
+        (["LL"], []),
+        (["MH", "LL"], ["MH"]),
+    ],
+)
+def test_bandit_order(lower, compare):
```

Comment:
Proposals:
1. Move auxiliary functions `level()` and `ratings()` to top level, i.e. outside test function `test_bandit_order()`
2. split test cases into a) expecting an `int` b) specifying a different input with better rating 
3. rename parameters appropriately, e.g. `input` and `expected` vs. `lower` and `higher`
4. remove case `assert False`